src/framework/preferences/js/PrefsEditor.js

```diff
@@ -391,13 +391,17 @@ var fluid_2_0 = fluid_2_0 || {};
             fluid.model.diff(userSelections.preferences, fluid.get(that.initialModel, ["preferences"]), stats);
 
             fluid.each(stats.changeMap, function (state, path) {
-                fluid.set(changedSelections, ["preferences", path], userSelections.preferences[path]);
+                fluid.set(changedPrefs, path, userSelections.preferences[path]);
```

Comment:
Do you think it would make sense if an integrator could decide what they wanted to save? It might make sense for some cases to only save the preferences and for others to save everything. One possible way to facilitate this would be to allow the path to be passed in.